iothub_client/src/iothubtransport_mqtt_common.c

```diff
@@ -2769,7 +2765,7 @@ static int UpdateMqttConnectionStateIfNeeded(PMQTTTRANSPORT_HANDLE_DATA transpor
             }
             else if (retry_action == RETRY_ACTION_STOP_RETRYING)
             {
-                // Set callback if retry expired
+                // send callback if retry expired
                 if (!transport_data->isRetryExpiredCallbackCalled)
```

Comment:
I think we talked about removing the isRetryExpiredCallbackCalled entirely. Was there a reason to keep it?

Reply:
failed a unit test, because every message sent would cause a connection notification. This was added outside mollie changes.